fix(dcgw): fix REP resource gateway and az overlap info (#6569)
Kong/developer.konghq.com: app/_how-tos/dedicated-cloud-gateways/aws-resource-endpoints.md

        * A [resource gateway](https://docs.aws.amazon.com/vpc-lattice/latest/ug/create-resource-gateway.html)
        * A [resource configuration group](https://docs.aws.amazon.com/vpc-lattice/latest/ug/create-resource-configuration.html)

        {:.warning}
        > **Important:** Your resource gateway's subnet must be in an Availability Zone (AZ) that overlaps with your Dedicated Cloud Gateway network's AZs.
        > If there's no AZ overlap, {{site.konnect_short_name}} can't establish the connection, and the resource endpoint config shows as `missing`, with no other indication of the cause.
        > Check which [AZs your Dedicated Cloud Gateway network supports](/konnect-platform/geos/#dedicated-cloud-gateways) before you create your resource gateway, and place it in a subnet within an overlapping AZ.

>         Copy and save the resource configuration ID and resource definition domain name for each resource configuration. {{site.konnect_short_name}} will use these to create a mapping of upstream domain names and resource configuration IDs.

Author: ```suggestion
        Copy and save the resource configuration ID for each resource configuration.
```

When using AWS Resource Endpoints with Dedicated Cloud Gateways, your resource gateway is the point of inbound traffic into your VPC for the resource you shared.
Traffic from your Dedicated Cloud Gateway arrives at your backend resources sourced from the resource gateway's own elastic network interface (ENI), using a normal IP address from the subnet you assigned to the resource gateway when you created it.

To allow this traffic, create a dedicated security group for your resource gateway, then reference that security group (not an IP range or prefix list) as the allowed source on your backend target's security group (for example, EC2 instances, Application Load Balancers, Network Load Balancers, or target Elastic Network Interfaces).

1. In AWS, navigate to your VPC console.
1. From the VPC sidebar, click **Security groups**.
1. Create a new security group (for example, `sg-resource-gateway`) and attach it only to your resource gateway.
1. Navigate to the security group for your backend target resource.
1. Add an inbound rule for the relevant port (for example, TCP/443).
1. In the **Source** field, select the dedicated resource gateway security group you created.

> 1. Repeat steps 3-6 for each backend resource that receives traffic through the resource gateway.

Reviewer: Ideally, we shouldn't be referencing step numbers since they can easily change. We also don't usually do something like "repeat the steps to create more of these" types of instructions - why are we doing it here? Can we just remove the last step?

 Also, would you expect users to create a new security group for every backend resource? I don't know the answer to that - that might be true.

> 1. Repeat steps 3-6 for each backend resource that receives traffic through the resource gateway.

Author: ```suggestion

Create a new security group for each backend resource that receives traffic through the resource gateway.
```

## Troubleshooting timeouts

If requests time out and your NLB shows no incoming traffic:

> * Verify the security group attached to your backend allows inbound traffic from the resource gateway's dedicated security group.

Reviewer: ```suggestion
* Verify that the security group attached to your backend allows inbound traffic from the resource gateway's dedicated security group.
```
"Verify the security group..." reads to me like I need to literally verify the security group.
